pkg/testframework/framework.go

```diff
@@ -109,16 +114,33 @@ func DeleteCompletely(getObject func() (metav1.Object, error), deleteObject func
 			return false, err
 		}
 
-		return obj.GetUID() != uid, nil
+		return getUID(obj) != uid, nil
 	})
 }
 
+func getUID(obj runtime.Object) types.UID {
+	configMap, ok := obj.(*corev1.ConfigMap)
```

Comment:
Wondering if passing a `runtime.Object` then casting it back is the best approach?

Reply:
yeah, just found out that the idiomatic way is to pass the runtime.Object into meta.Accessor which will return meta.Object

Reply:
Using meta.Accessor is also more safe cause we can get error if passed object is not supported.